### PR TITLE
Fixed center frequency hang issue

### PR DESCRIPTION
**Pull request by AA6KL**
This fixes https://github.com/hugen79/NanoVNA-H/issues/25

### CODE REVIEW

File: plot.c [hunk 1]
```diff
@@ -1464,6 +1464,33 @@ void request_to_draw_cells_behind_numeric_input(void)
...
+             (int)(freq / 1000000),
+             (int)((freq / 1000) % 1000),
+             (int)(freq % 1000));
+  }
+  l = strlen(buf);
+  for (i=l; i<len-1; i++) {
+    buf[i]=' ';
+  }
+  buf[len-1]=0;
+  return(0);
```
Comment (Pmax65) [inline plot.c]: Why does this void function return 0?

Comment (hugen79) [reply plot.c]: This function does not need to return a value, I will fix it after the merge.

> Why does this void function return 0?

This function does not need to return a value, I will fix it after the merge.

**Comment by Pmax65** [comment]
Hi Hugen,
please take care to apply this modification to the nanoVNA master too.
Because, even if it doesn't show any apparent malfunction, this issue is present in that firmware too.
This should be reported to edy555 and qrp77 in case they don't follows this discussion.

Have a great day.

Massimo

By the way, do you have a public repository for your pre-compiled firmwares?
I'm a really rookie in this community.

**Comment by hugen79** [comment]
> Hi Hugen,
> please take care to apply this modification to the nanoVNA master too.
> Because, even if it doesn't show any apparent malfunction, this issue is present in that firmware too.
> This should be reported to edy555 and qrp77 in case they don't follow this discussion.
> 
> Have a great day.
> 
> Massimo
> 
> By the way, do you have a public repository for your pre-compiled firmwares?
> I'm a really rookie in this community.

Hi Massimo
Thank you for reminding, this problem may be an error when I created the AA version, I will check the relevant code later.

hugen

**Comment by Pmax65** [comment]
I apologize, sometimes I write things that are obvious for others, but I'm really a newbie in this community, and I know very little about how it works.

I'm still working on the RF analog issues to see what we can do for get it better.
As soon as I'll have something useful I inform you.

In the meantime have great days.

Massimo

**Pull request merged**